source/stylesheets/refills/_vertical-tabs.scss

```diff
@@ -103,16 +93,16 @@
   }
 
   .vertical-tab-accordion-heading {
-    background-color: $tab-inactive-color;
+    background-color: $tab-inactive-background;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets